packages/playground/cli/README.md

```diff
@@ -95,6 +95,9 @@ The `server` command supports the following optional arguments:
 -   `--skip-sqlite-setup`: Do not set up the SQLite database integration.
 -   `--quiet`: Do not output logs and progress messages.
 -   `--debug`: Print the PHP error log if an error occurs during boot.
+-   `--follow-symlinks`: Allow Playground to follow symlinks by automatically mounting symlinked directories and files encountered in mounted directories. ⚠️ Warning: Following symlinks will expose files outside mounted directories to Playground and could be a security risk.
+-   `--experimentalMultiWorker`: Enables experimental multi-worker support. It needs JSPI and a /wordpress directory on a real filesystem. You can pass a positive number to use a specific number of workers, otherwise, it defaults to the number of CPUs minus one.
```

Comment:
Let's rename that to `--experimental-multi-worker` in CLI to keep it consistent with other flags